templates/webhook.service.j2

```diff
@@ -13,6 +13,9 @@ ExecStart={{ github_webhook_script_dir }}/server.py \
     --port={{ github_webhook_port }} \
     --repo-url={{ github_webhook_repo_url | mandatory }} \
     --repo-branch={{ github_webhook_repo_branch | mandatory }} \
+{% for instance in github_webhook_instances if instance != inventory_hostname %}
+    --broadcast-instances='{{ instance }}.wg:{{ github_webhook_port }}' \
```

Comment:
Hardcoding the `.wg` suffix here seems unnecessary, since we could have a case where we use public endpoint.

Reply:
Honestly, it should be whole URLs.

Reply:
You don't have to use public endpoint, just HTTP over wireguard. I'm just talking how the URL is provided.